analyzers/src/SonarAnalyzer.Common/Rules/Utilities/UtilityAnalyzerBase.cs

```diff
@@ -76,9 +76,11 @@ protected void ReadParameters(SonarAnalysisContext context, CompilationAnalysisC
     public abstract class UtilityAnalyzerBase<TMessage> : UtilityAnalyzerBase
         where TMessage : IMessage, new()
     {
+        private const int TokenCountThreshold = 10_000;
```

Comment:
We should have some UT or IT for this. This trick can help with that:
https://github.com/SonarSource/sonar-dotnet/blob/f6feccca65fb24d565fd3a9a614cc66b69125fdc/analyzers/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs#L5240